_posts/2.5/2021-06-09-new.md

```diff
@@ -61,6 +61,14 @@ You can now pin a webcam so it always stays visible.  This is useful if one on o
 
 Private guest lobby message, display position in waiting line
 
+### Improvements to Endpoint API 
+
+#### New endpoint
+
+A new API endpoint was added: `/insertDocument`. Now, it is possible to send a batch of documents with all the common presentation parameters such as `current`, `downloadable` and `removable`. 
+
+All the presentations are sent via payload (the body of the request) such as when preuploading documents in `/create` endpoint, whether by a link to download the file or by `base64` encoded file. See the [API documentation](../dev/api.html#insertdocument) for more information
```

Comment:
```suggestion
All the presentations are sent via payload (the body of the request) such as when preuploading documents in `/create` endpoint, whether by a link to download the file or by `base64` encoded file. See the [API documentation](/dev/api.html#insertdocument) for more information
```

or full link (I think jekyll will have troubles with `../` but I have not actually ran it, just proofreading